quarkus/runtime/src/main/java/org/keycloak/connections/jpa/QuarkusJpaConnectionProviderFactory.java

```diff
@@ -116,14 +120,15 @@ public void init(Config.Scope config) {
     }
 
     private void addSpecificNamedQueries(KeycloakSession session, Connection connection) {
-        LiquibaseConnectionProvider liquibaseProvider = session.getProvider(LiquibaseConnectionProvider.class);
-        EntityManager em = null;
+        SessionFactoryImplementor sfi = emf.unwrap(SessionFactoryImplementor.class);
```

Comment:
OK, everything is working for me now. Just a minor thing, this line and the three {{org.hibernate}} imports at the beginning of the file are not needed now.